app/views/job_templates/run_openscap_remediation_-_ansible_default.erb

```diff
@@ -22,7 +22,14 @@ require:
 ---
 - hosts: all
   tasks:
-<%= indent(4) { input('tasks') } -%>
+<%= snippet('check_bootc_status') %>
+    - name: Enable bootc overlay
+      shell:
+        cmd: 'bootc usr-overlay'
+      register: out
+      ignore_errors: true
+      when: is_bootc_host
+    - debug: var=out
```

Comment:
Would it make sense to print this only conditionally?

Reply:
Perhaps, I included it because the Foreman Ansible shell command template includes the same debug output.
What would it look like as a conditional?